tests/console/docker_runc.pm

```diff
@@ -46,11 +53,12 @@ sub run {
 
     # installation of runc package
     record_info 'Test #1', 'Test: Installation';
-    if (is_caasp('DVD') && !check_var('SYSTEM_ROLE', 'plain')) {
-        # runC should be pre-installed in MicroOS
-        die "runC is not pre-installed." if script_run("zypper se -x --provides -i runc | grep runc");
+    if (is_caasp) {
+        # runC should be pre-installed
+        die "runC is not pre-installed." if script_run("zypper se -x --provides -i $runc | grep runc");
```

Comment:
I tried manually on MicroOS 3.0 (alpha1 plain role) - I think this will fail, or am I wrong?
```
dhcp225:~ # zypper se -x --provides -i docker-runc
Loading repository data...
Reading installed packages...
No matching items found.
 
dhcp225:~ # which docker-runc
/usr/bin/docker-runc
 
dhcp225:~ # rpm -qf /usr/bin/docker-runc
runc-0.1.1+gitr2819_50a19c6-1.2.x86_64
```

Reply:
this is a product bug then.